analysis/evaluate_policies.py

```diff
@@ -49,3 +45,12 @@ def run_analysis(
     figure = visualize_analysis(instances, policies, smooth_curve)
 
     return figure
+
+
+if __name__ == "__main__":
+    run_analysis(
+        policies=["RandomRolloutPolicy", "SwapAllPolicy"],
+        sample_size=1000,
+        number_of_clusters=100,
+        verbose=True,
+    )
```

Comment:
Import the input variables from globals? if not, remove them from globals

Reply:
The variables in the globals files are used in the tests. The tests take forever if we need to change them to these values